appinventor/docs/reference/components/userinterface.html

```diff
@@ -792,30 +792,30 @@ <h2 id="ListPicker">ListPicker</h2>
 
 <h2 id="ListView">ListView</h2>
 
-<p><p>This is a visible component that allows to place a list of text elements in your Screen to display. <br> The list can be set using the ElementsFromString property or using the Elements block in the blocks editor. <br> Warning: This component will not work correctly on Screens that are scrollable. </p></p>
+<p><p>This is a visible component that allows a list of text elements in your Screen to be displayed. <br> The list can be set using the ElementsFromString property or using the Elements block in the blocks editor. <br> Warning: This component will not work correctly on Screens that are scrollable. </p></p>
 
 <h3>Properties</h3>
 <dl>
   <dt><code>BackgroundColor</code></dt>
-  <dd>The color of the listview background.</dd>
+  <dd>The color of the ListView background.</dd>
   <dt><code>Elements</code></dt>
   <dd>List of text elements to build your list.</dd>
   <dt><code>ElementsFromString</code></dt>
   <dd>Build a list with a series of text elements separated by commas such as: Cheese,Fruit,Bacon,Radish. Each word before the comma will be an element in the list.</dd>
   <dt><code>Height</code></dt>
-  <dd>Determines the height of the list on the view.</dd>
+  <dd>Determines the height of the list on the view (y-size).</dd>
   <dt><code>Selection</code></dt>
   <dd>Returns the text last selected in the ListView.</dd>
   <dt><code>SelectionIndex</code></dt>
   <dd>The index of the currently selected item, starting at 1.  If no item is selected, the value will be 0.  If an attempt is made to set this to a number less than 1 or greater than the number of items in the ListView, SelectionIndex will be set to 0, and Selection will be set to the empty text.</dd>
   <dt><code>ShowFilterBar</code></dt>
-  <dd>Sets visibility of ShowFilterBar. True will show the bar, False will hide it.</dd>
+  <dd>Sets visibility of the filter bar. True will show the bar, false will hide it.</dd>
```

Comment:
ShowFilterBar as a noun, and also False as a noun (value). I would not change these.